decidim-core/app/packs/src/decidim/input_multiple_mentions.js

```diff
@@ -1,196 +1,85 @@
-      <span class="author__avatar"><img src="${item.original.avatarUrl}" alt="author-avatar"></span>
-        <strong>${item.original.nickname}</strong>
-        <small>${item.original.name}</small>
-        ${svg}
-        ${disabledElementMessage}
-      </div>`;
     }
   });
 
-  let setupEvents = function($element) {
-    // DOM manipulation
-    $element.on("focusin", (event) => {
-      // Set the parent container relative to the current element
-      tribute.menuContainer = event.target.parentNode;
-    });
-    $element.on("focusout", (event) => {
-      let $parent = $(event.target).parent();
-
-      if ($parent.hasClass("is-active")) {
-        $parent.removeClass("is-active");
-      }
-    });
-    $element.on("input", (event) => {
-      let $parent = $(event.target).parent();
+  $searchInput.on("selection", (event) => {
+    const feedback = event.detail;
+    const selection = feedback.selection;
+    const id = selection.value.id;
+    if (selected.length >= 9 || selection.value.directMessagesEnabled === "false") {
+      return;
+    }
 
-      if (tribute.isActive) {
-        // We need to move the container to the wrapper selected
-        let $tribute = $(".tribute-container");
-        $tribute.appendTo($parent);
-        // Remove the inline styles, relative to absolute positioning
-        $tribute.removeAttr("style");
-        // Parent adaptation
-        $parent.addClass("is-active");
-      } else {
-        $parent.removeClass("is-active");
+    $selectedItems.append(`
+      <li>
+        <input type="hidden" name="${options.name}" value="${selection.value.id}">
+        <img src="${selection.value.avatarUrl}" alt="${selection.value.name}">
+        <b>${selection.value.name}</b>
+        <button class="float-right" data-remove=${id} tabindex="0" aria-controls="0" aria-label="Close" role="tab">&times;</button>
+      </li>
+    `);
+
+    autoComplete.setInput("");
+    selected.push(id);
+
+    $selectedItems.find(`*[data-remove="${id}"]`).on("keypress click", (evt) => {
+      const target = evt.target.parentNode;
+      if (target.tagName === "LI") {
+        selected = selected.filter((identifier) => identifier !== id)
+        target.remove();
       }
-    });
-  };
+    })
+  })
 
-  let setupRecipientEvents = function($element) {
-    // Allow delete with click on element in recipients list
-    $element.on("click", (event) => {
-      let $target = event.target.parentNode;
-      if ($target.tagName === "LABEL") {
-        deleteRecipient($target);
-      }
-    });
-    // Allow delete with keypress on element in recipients list
-    $element.on("keypress", (event) => {
-      let $target = event.target.parentNode;
-      if ($target.tagName === "LABEL") {
-        deleteRecipient($target);
-      }
-    });
-  };
+  // Stop input field from bubbling open and close events to parent elements,
+  // because foundation closes modal from these events.
+  $("#autocomplete").on("open close", (event) => {
```

Comment:
I think this should be moved to the base `AutoComplete` class in order for the same fix to apply everywhere autocomplete is used.